nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,192 @@
+        Defaults to "Group help".
+    group_embed_description: Optional[:class:`str`]
+        The description for the embed which gets sent by invoking `help <group>`.
+    group_embed_color: Optional[:class:`int`]
+        The color for the embed which gets sent by invoking `help <group>`.
+        Defaults to default_color.
+    cog_embed_title: :class:`str`
+        The title for the embed which gets sent by invoking `help <cog>`.
+        Defaults to "Cog help".
+    cog_embed_description: Optional[:class:`str`]
+        The description for the embed which gets sent by invoking `help <cog>`.
+    cog_embed_color: Optional[:class:`int`]
+        The color for the embed which gets sent by invoking `help <cog>`.
+        Defaults to default_color.
+    """
+
+    def __init__(
+        self,
+        *,
+        dm_help: bool = False,
+        default_color: int = 0xFFFFFF,
+        main_embed_title: str = "Overview about Cogs, Groups and Commands",
+        main_embed_description: Optional[str] = None,
+        main_embed_color: Optional[int] = None,
+        command_embed_title: str = "Command help",
+        command_embed_description: str = "If a parameter is surrounded by `<>`, it is a required parameter. \\\\n If a parameter is surrounded by `[]`, it is an optional parameter.",
+        command_embed_color: Optional[int] = None,
+        group_embed_title: str = "Group help",
+        group_embed_description: Optional[str] = None,
+        group_embed_color: Optional[int] = None,
+        cog_embed_title: str = "Cog help",
+        cog_embed_description: Optional[str] = None,
+        cog_embed_color: Optional[int] = None,
+    ):
+
+        self.dm_help = dm_help
+        self.default_color = default_color
+
+        self.main_embed_title = main_embed_title
+        self.main_embed_description = main_embed_description
+        self.main_embed_color = main_embed_color or self.default_color
+
+        self.command_embed_title = command_embed_title
+        self.command_embed_description = command_embed_description
+        self.command_embed_color = command_embed_color or self.default_color
+
+        self.group_embed_title = group_embed_title
+        self.group_embed_description = group_embed_description
+        self.group_embed_color = group_embed_color or self.default_color
+
+        self.cog_embed_title = cog_embed_title
+        self.cog_embed_description = cog_embed_description
+        self.cog_embed_color = cog_embed_color or self.default_color
+
+        super().__init__()
+
+    @staticmethod
+    def determine_group_or_command(obj: Union[commands.Command, commands.Group]):
+        return f"`{obj.name}[group]`" if isinstance(obj, commands.Group) else f"`{obj.name}`"
+
+    async def send_embed(self, emb: Embed):
+        return (
+            await self.context.author.send(embed=emb)
+            if self.dm_help
+            else await self.context.send(embed=emb)
+        )
```

Comment:
```suggestion
        if self.dm_help:
            return await self.context.author.send(embed=emb)
        return await self.context.send(embed=emb)
```
imo, readability suffers from this ternary

Reply:
Really, this is the exact purpose for `get_destination`.

This is what should be used:

https://github.com/nextcord/nextcord/blob/0c7f0ebaaf06d9ec14f2669c05a458be058aa54a/nextcord/ext/commands/help.py#L1257-L1264